history/management/commands/pull_deposits.py

```diff
@@ -35,6 +35,4 @@ def handle(self, *args, **options):
             d.status = status
             d.created_on = created_on
             d.modified_on = created_on
```

Comment:
I think @owocki intention was to capture the time of creation on the poloniex server with this. 
A proposal could be we add `deposit_created_on` to `Deposit` model and use `created_on`  for the creation of the model in our database.
In short:
`deposit_created_on` - time of creation at poloniex server
`created_on` - time of saving in our database